### DISCUSSION TITLE
Donate and feel good.

### DISCUSSION THREAD

**Discussion by alistairgd** [Rem0o/FanControl.Releases#1253] [General]
If like me you just spent a fair bit of your disposable income on building a new PC, and you are enjoying the full control that this app gives you to balance noise, power usage, and safeguard your components - think about donating just a few dollars to support further development and just to say thanks for such a cool tool.

:o)

[Donate Here](https://www.paypal.com/donate/?cmd=_donations&business=N4JPSTUQHRJM8&currency_code=USD&source=url&item_name=Fan+Control)

**Comment by jeffreybasurto** [comment]
I wanted to donate, but 
![image](https://user-images.githubusercontent.com/300655/204120805-bc10e98c-c759-4f3a-bc2c-5d1fb715b27c.png)
:(

**Reply by octav59** [reply]
Just made a donation.
The perfect tool for my new rig.
Thanks, Remi, genius software.

**Reply by Rem0o** [reply]
@jeffreybasurto you can take a look at the sponsor section of github as an alternative.

**Comment by Dreams1010** [comment]
Thanks for this software! Works really well!
Merci!

**Comment by EvilRunner** [comment]
Mix Curve, yes !!!! Exactly what i'm locking for, thanks and take my little donation ;)
